communitiesuk/funding-service-design-assessment
# FS-3843: Display team metrics after filters applied

[https://dluhcdigital.atlassian.net/browse/FS-3843](https://dluhcdigital.atlassian.net/browse/FS-3843)


### Change description
Amended the Team stats so it uses all applications meta data to produce the stats meaning the stats do not change when a filter is applied



### How to test
- Log into the assessment tool, open a fund round, view applications
- Verify the team stats appear
- Apply a filter
- Verify that the stats have not changed
- Repeat for all filters and search


### Screenshots of UI changes (if applicable)
Team stats on first load of the page
![image](https://github.com/user-attachments/assets/8b9bfa59-0c1a-48c9-b883-9a8bb3fdc47e)

Team stats when a filter is applied
![image](https://github.com/user-attachments/assets/ef03f849-4d5d-4577-8805-333d2b6a953e)
